app/views/products/_form.html.erb

```diff
@@ -50,7 +57,7 @@
 
   <div class="field">
     <%= form.label :category_id %>
-    <%= form.select :category_id, options_for_select(Category.all.collect { | category | [category.name, category.id] }, @product.category_id), id: :category_id, prompt: 'choose category' %>
+    <%= form.select :category_id, options_for_select(Category.all.collect { | category | [category.name, category.id] }, @product.category_id), id: :category_id, prompt: 'Select one' %>
```

Comment:
Use `include_blank` here
let me know difference between `prompt` and `include_blank`